### PR TITLE
Cleanup abilities/permissions

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Add host/application instance/database instance cleanup abilities.
The `Clean up` is only available when the user has the proper abilities attached.

Even though we have a lot of new lines, almost everything is boilerplate and `userAbilities` movement in the frontend.

### CODE REVIEW

**Review by EMaksy — APPROVED** [review]
Hey man LGTM! Great work!

File: assets/js/pages/HostsList/HostsList.test.jsx
Comment (arbulu89) [reply assets/js/pages/HostsList/HostsList.test.jsx:214]: @EMaksy Yeah...
This is true.
I like the idea of having a constant for this, so we can reuse in tests and storybook.
As we have some already going work in parallel, do you mind if we wait until all the frontend PRs are merged to apply this change?
This way, we don't have any sort of conflicts.
Basically, we do a big search/replace once we have all the features done

Comment (EMaksy) [reply assets/js/pages/HostsList/HostsList.test.jsx:214]: Yeah, sure!
We could even define some more user ability roles, so in future if something changes we need to change it only at one place and not everywhere.

But yeah, let's finish up everything else first!

**Review by CDimonaco — APPROVED** [review]
LGTM

Just a nitpick, if you want you can add a controller test that accepts the request when the abilities are present.

Other then all:all ability, we have that in the setup function, but I think it's valuable, pretty easy to create a user on demand with the right ability and test it.

It serves also as docs purpose about the controller usage

Wdyt?

**Pull request merged**
